netanalysis/traffic/analysis/find_anomalies.py

```diff
@@ -265,7 +266,7 @@ def main(args):
 
     try:
         all_disruptions = find_all_disruptions(
-            repo, repo.list_regions(), product_id_list)  # type: List[RegionDisruption]
+            repo, repo.list_regions(), product_id_list)  # type: List[model.RegionDisruption]
```

Comment:
make this type explicit if possible

all_disruptions: List[model.RegionDisruption] = find_all_disruptions(repo, repo.list_regions(), product_id_list)

Reply:
Done